<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>camerawidget</class>
 <widget class="QWidget" name="camerawidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1116</width>
    <height>565</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>12</pointsize>
   </font>
  </property>
  <property name="focusPolicy">
   <enum>Qt::NoFocus</enum>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>770</width>
        <height>545</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="lbImage">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>无图像</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>320</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>320</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <widget class="QPushButton" name="pbBack">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>20</y>
        <width>80</width>
        <height>64</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton:hover { background-color: rgb(200, 200, 200); border-radius: 3px; color: rgb(255, 255, 255); }
QPushButton:focus { background-color: rgb(100, 100, 100); border:5px; }  </string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>ico/back.png</normaloff>ico/back.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>64</height>
       </size>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pbSave">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>20</y>
        <width>80</width>
        <height>64</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton:hover { background-color: rgb(200, 200, 200); border-radius: 3px; color: rgb(255, 255, 255); }
QPushButton:focus { background-color: rgb(100, 100, 100); border:5px; }  </string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>ico/save.png</normaloff>ico/save.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>64</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pbPause">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>150</y>
        <width>80</width>
        <height>40</height>
       </rect>
      </property>
      <property name="text">
       <string>冻结</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pbVideo">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>280</y>
        <width>80</width>
        <height>64</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QPushButton:hover { background-color: rgb(200, 200, 200); border-radius: 3px; color: rgb(255, 255, 255); }
QPushButton:focus { background-color: rgb(100, 100, 100); border:5px; }  </string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>ico/video.png</normaloff>ico/video.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>64</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>90</y>
        <width>54</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>返回</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbVideo">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>350</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>开始录像</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>90</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>抓取图片</string>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>230</y>
        <width>271</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>370</y>
        <width>271</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>250</y>
        <width>54</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>录像</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>280</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>大小：</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbVideoSize">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>280</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>0.000</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>310</y>
        <width>41</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>时长：</string>
      </property>
     </widget>
     <widget class="QLabel" name="lbVideoTime">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>310</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>00:00:00</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pbFullScreen">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>20</y>
        <width>80</width>
        <height>64</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>ico/fullscreen.png</normaloff>ico/fullscreen.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>80</width>
        <height>64</height>
       </size>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="lbScreen">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>90</y>
        <width>71</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>全屏显示</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="cbAutoSave">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>250</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>自动保存</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pbBack</tabstop>
  <tabstop>pbSave</tabstop>
  <tabstop>pbPause</tabstop>
  <tabstop>pbVideo</tabstop>
  <tabstop>widget</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
